frontend/src/index.css

```diff
@@ -1,194 +1,104 @@
-@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wdth,wght@0,62.5..100,100..900;1,62.5..100,100..900&display=swap');
```

Comment:
Why is this removed ?

Please verify all your changes. Make sure you are not removing any essential code.